src/build/optimize-pipe.ts

```diff
@@ -10,9 +10,51 @@
 
 import * as stream from 'stream';
 
+import * as gulpif from 'gulp-if';
```

Comment:
Both gulp-if and minimatch are third_party dependencies, and sorted. Not sure why it needs a newline.